mod.json

```diff
@@ -48,7 +48,8 @@
 			"UrbanExplosionsFix": false,
 			"WeakAgainstMeleeFix": true,
 			"WeaponTooltip": true,
-			"DamageReductionInCombatHud": false
+			"DamageReductionInCombatHud": false,
+      "DeathChanceStat": true
```

Comment:
Looks like you're using spaces while the rest of the file is using tabs.

Reply:
fixed